### PR TITLE
Added command to move particular facilities to another district

### PR DESCRIPTION
**Pull request by kkrampa**

### CODE REVIEW

File: logistics_project/apps/tanzania/management/commands/tz_move_facilities.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+            default=False,
+            help="Just print out test information but don't actually migrate anything"
+        ),
+    )
+
+    def handle(self, *args, **options):
+        moving_map = {
+            'Chemba DC': [
+                'D35907',
+                'DM 520138',
```
Comment (kkrampa) [inline logistics_project/apps/tanzania/management/commands/tz_move_facilities.py]: It's not a typo

File: logistics_project/apps/tanzania/management/commands/tz_move_facilities.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+                'DM 520140',
```
Comment (kkrampa) [inline logistics_project/apps/tanzania/management/commands/tz_move_facilities.py]: same here

File: logistics_project/apps/tanzania/management/commands/tz_move_facilities.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+                'DM 520146',
```
Comment (kkrampa) [inline logistics_project/apps/tanzania/management/commands/tz_move_facilities.py]: and here

**Comment by czue** [comment]
@kkrampa I think we can just use https://github.com/dimagi/logistics/blob/tz-master/logistics_project/apps/tanzania/management/commands/tz_move_locations.py as long as we generate the right file, right? we need to use the special command since it also has to go back and fix the warehouse data

**Comment by kkrampa** [comment]
@czue Updated.

I needed to update a little bit this command because site code for district that we need contains spaces.

command ./manage.py tz_move_locations apps/tanzania/management/commands/resource/facility_migration_1.csv --byname --strict

**Pull request merged**